kafka-store/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -83,25 +81,22 @@
   // messages with this key
   private final K noopKey;
   private volatile long lastWrittenOffset = -1L;
-  private final SchemaRegistryConfig config;
+  private final KafkaStoreConfig config;
 
-  public KafkaStore(SchemaRegistryConfig config,
+  public KafkaStore(KafkaStoreConfig config,
                     StoreUpdateHandler<K, V> storeUpdateHandler,
                     Serializer<K, V> serializer,
                     Store<K, V> localStore,
-                    K noopKey) {
+                    K noopKey,
+                    boolean isZkSecurityEnabled) {
     this.kafkaClusterZkUrl =
-        config.getString(SchemaRegistryConfig.KAFKASTORE_CONNECTION_URL_CONFIG);
-    this.topic = config.getString(SchemaRegistryConfig.KAFKASTORE_TOPIC_CONFIG);
+        config.getString(KafkaStoreConfig.KAFKASTORE_CONNECTION_URL_CONFIG);
+    this.topic = config.getString(KafkaStoreConfig.KAFKASTORE_TOPIC_CONFIG);
     this.desiredReplicationFactor =
-        config.getInt(SchemaRegistryConfig.KAFKASTORE_TOPIC_REPLICATION_FACTOR_CONFIG);
-    int port = KafkaSchemaRegistry.getPortForIdentity(config.getInt(SchemaRegistryConfig.PORT_CONFIG),
-            config.getList(RestConfig.LISTENERS_CONFIG));
-    this.groupId = String.format("schema-registry-%s-%d",
```

Comment:
This is where we seem to have lost some unique information about the schema registry instance. Since this is used as the consumer group ID, just using `"test-group"` in the other part of the patch isn't ideal. In practice, you can probably get away with this because we use the new consumer in "simple" consumer mode and don't commit offsets. However, this makes some logs much harder to use since they don't have unique identifying info. We probably want to maintain the group ID if possible.